ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-master.yaml

```diff
@@ -49,6 +49,10 @@ tests:
   commands: make vet
   container:
     from: src
+- as: crd-gen-check
```

Comment:
It's going to check all of the generated output, right? Maybe a better name would be just "generate-check"?